test/install_test.py

```diff
@@ -18,6 +18,11 @@ def testResolve(self):
         artifacts = _resolve_artifacts([junit])
         self.assertEqual(len(artifacts), 1)
 
+    def testResolve2(self):
```

Comment:
Can you please give the test method a more meaningful name? Like testResolveArtifactWithUmlautsInPom or so?